app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt

```diff
@@ -67,17 +62,17 @@ class FileRepository(private val context: Context) {
             FileOutputStream(File(pathName))
         }
 
-    fun takeScreenShot(bitmap: Bitmap): File? {
-        val date = Date()
-        val format = DateFormat.format("MM-dd-yyyy_hh:mm:ss", date)
+    fun takeScreenShot(bitmap: Bitmap, name: String? = null): File? {
+        val fileName = name ?: Date().toString()
         try {
-            val mainDir =
-                File(context.getExternalFilesDir(Environment.DIRECTORY_PICTURES), context.getString(R.string.app_name))
+            val mainDir = File(
+                context.getExternalFilesDir(Environment.DIRECTORY_PICTURES),
```

Comment:
Please, make sure it's working on different OS versions.
I assume there are difference between OS Android.Q and other lower versions

Reply:
it was already done for SecurityKey so i presume it is working (should =))